### ISSUE TITLE
Make cutis hyperelastica a deprecated synonym

### ISSUE THREAD

**Issue by nicolevasilevsky** [monarch-initiative/mondo#137]
_Pasted email thread below_

You mean the "cutis hyperelastica" synonym? Nicole can you follow up on this, add a ticket, and remember we want to source all claims with a PMID or similar

On 12 Apr 2018, at 9:06, Julie McMurry wrote:

Please make this so for cutis laxica so I can give our points back on the table.

On Thu, Apr 12, 2018 at 8:39 AM, Peter Robinson <Peter.Robinson@jax.org> wrote:
​This would do it, I think. Does it correspond to something in RO?

Peter Robinson

From: Chris Mungall 
Sent: Thursday, April 12, 2018 11:30 AM
To: Melissa Haendel
Cc: Peter Robinson; ; Nicole Vasilevsky
Subject: Re: MONDO: notion of obsoleted synonym?
 
We have a DEPRECATED synonym type. Examples

[Term]
name: intellectual disability
synonym: "mental retardation" RELATED DEPRECATED []

[Term]
name: thymic carcinoma
synonym: "Thymoma Type C" EXACT DEPRECATED [NCIT:C7569]
synonym: "type C thymoma" EXACT DEPRECATED [MONDO:0003048, NCIT:C7569]

[Term]
name: pemphigoid gestationis
synonym: "Gestational herpes" EXACT DEPRECATED [DOID:14482, SCTID:156129002, SCTID:270959002]
synonym: "Herpes gestationis" EXACT DEPRECATED [DOID:14482, MTHICD9_2006:646.8]
synonym: "Herpes gestationis (disorder)" EXACT DEPRECATED [DOID:14482, SCTID:86081009]
synonym: "Herpes gestationis NOS (disorder)" EXACT DEPRECATED [DOID:14482, SCTID:199131000]
synonym: "Herpes gestationis unspecified (disorder)" EXACT DEPRECATED [DOID:14482, SCTID:199126002]

[Term]
name: alcohol withdrawal delirium
synonym: "Delirium Tremens" RELATED DEPRECATED [MESH:D000430]
synonym: "Delirium Tremens, Alcohol Withdrawal Induced" RELATED DEPRECATED [MESH:D000430]
synonym: "Delirium, Alcohol Withdrawal" RELATED DEPRECATED [MESH:D000430]

[Term]
name: myoclonus and ataxia
synonym: "Ramsay Hunt Syndrome" RELATED DEPRECATED [https://en.wikipedia.org/wiki/Ramsay_Hunt_syndrome, OMIM:159700]
synonym: "Ramsay Hunt syndrome type 1" RELATED DEPRECATED [https://en.wikipedia.org/wiki/Ramsay_Hunt_syndrome, OMIM:159700]

[Term]
name: neurofibromatosis type 3
synonym: "neurilemmomatosis" RELATED DEPRECATED [DOID:3204, https://en.wikipedia.org/wiki/Schwannomatosis]
synonym: "neurinoma" RELATED DEPRECATED [https://en.wikipedia.org/wiki/Schwannomatosis]

[Term]
name: neurodegeneration with brain iron accumulation 2a
synonym: "Hunter-Carpenter-McDonald syndrome" RELATED DEPRECATED [Orphanet:2174]

[Term]
name: myofibrillar myopathy 1
synonym: "IBM1" RELATED DEPRECATED [https://en.wikipedia.org/wiki/Hereditary_inclusion_body_myopathy]

[Term]
name: Ehlers-Danlos syndrome, kyphoscoliotic type
synonym: "Nevo Syndrome" EXACT DEPRECATED [OMIM:225400, PMID:19760654]

[Term]
name: ameloblastoma
synonym: "adamantinoma" RELATED DEPRECATED [https://en.wikipedia.org/wiki/Adamantinoma]

[Term]
name: autosomal dominant nonsyndromic deafness
synonym: "autosomal dominant nonsyndromic hearing impairment" EXACT DEPRECATED [https://ghr.nlm.nih.gov/condition/nonsyndromic-hearing-loss]

[Term]
name: autosomal dominant limb-girdle muscular dystrophy type 1D (DNAJB6)
synonym: "autosomal dominant limb-girdle muscular dystrophy type 1E" RELATED DEPRECATED [DOID:0110305]
synonym: "LGMD1E" RELATED DEPRECATED [https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4021627/, MONDO:Lexical, OMIM:603511]
synonym: "LGMD1E (Bushby and Beckmann, 2003)" RELATED DEPRECATED [https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4021627/, OMIM:603511]
synonym: "muscular dystrophy limb-girdle type 1E" RELATED DEPRECATED [DOID:0110305]
synonym: "MUSCULAR DYSTROPHY, LIMB-GIRDLE, TYPE 1E" RELATED DEPRECATED [MONDO:Lexical, OMIM:603511]
synonym: "MUSCULAR DYSTROPHY, LIMB-GIRDLE, TYPE 1E; LGMD1E" RELATED DEPRECATED [OMIM:603511]

[Term]
name: hypophosphatemic rickets
synonym: "Rickets, Vitamin D Resistant" RELATED DEPRECATED [MESH:D063730]
synonym: "Rickets, Vitamin D-Resistant" RELATED DEPRECATED [MESH:D063730]
synonym: "Vitamin D-Resistant Rickets" RELATED DEPRECATED [MESH:D063730]
On 12 Apr 2018, at 6:50, Melissa Haendel wrote:

I agree that this will be important for MONDO and will distinguish it. I don’t care about the label, but lets put the property in RO or IAO so it can be used by others.

On Apr 12, 2018, at 3:38 AM, Peter Robinson wrote:

I think we need a label for legacy terminology that we recommend should not be used anymore but we are retaining in order to have a link to old literature. We had this for HPO as well, but for some reason we could not agree on the label. has_deprecated_synonym might get us half way there?

Peter Robinson

From: Julie McMurry 
Sent: Thursday, April 12, 2018 2:43 AM
To: Chris Mungall; Peter Robinson; Nicole Vasilevsky; Melissa Haendel
Subject: MONDO: notion of obsoleted synonym?
 
Peter said that cutis elastica and cutis hyperelastica were not great synonyms and should be obsoleted. Do we have a way to do this in MONDO? If we go with peters assessment as-is, we'll have two false positives for MONDO. I would like to avoid, but it is not a big deal.

J

**Comment by nicolevasilevsky** [comment]
looks like this is done. closing.
